routers/comments.py

```
@@ -0,0 +1,75 @@
+from datetime import datetime, timezone
+from typing import List
```

Ruff (0.14.11)

`typing.List` is deprecated, use `list` instead

(UP035)